packages/link-list/CHANGELOG.md

```diff
@@ -32,6 +33,11 @@
 
 ## Release History
 
+### v3.0.0
+
+- Move react children location
```

Comment:
You got so much space here, good place to describe this a bit more in detail. Why, from where, etc?